docs/source/index.rst

```diff
@@ -9,16 +9,16 @@
 
 *A particle-in-cell code for GPGPUs*
 
-PIConGPU is a fully relativistic, many GPGPU, 3D3V particle-in-cell (PIC) code.
+PIConGPU is a fully relativistic, manycore, 3D3V particle-in-cell (PIC) code.
```

Comment:
please keep the title, this is intentional a play of words.
new title is discussed in #2250

Reply:
Ah true, I see. It still changes meaning but I am fine with that.